### PR TITLE
Package upgrades

### PR DESCRIPTION
**Pull request by AkimaLunar**
Routine package upgrades: all dev dependencies (lint, storybook, etc), except `typescript` and `@griffel/react`.

### CODE REVIEW

**Review by Mitch-At-Work — APPROVED** [review]

**Comment by Mitch-At-Work** [comment]
Change files needed but otherwise good to go

File: components/action-list-tile/package.json [hunk 1]
```diff
@@ -26,11 +26,11 @@
...
     "@microsoft/arbutus.use-space-styles": "1.0.5",
     "@fluentui/react-theme": "^9.1.8",
-    "@griffel/react": "^1.5.7",
+    "@griffel/react": "^1.5.17",
     "@reach/visually-hidden": "^0.18.0"
   },
   "devDependencies": {
     "@microsoft/arbutus.ts": "0.0.2",
-    "typescript": "^5.1.3"
+    "typescript": "^5.2.2"
```
Comment (brandonthomas) [inline components/action-list-tile/package.json:34]: Do we want to look into hoisting the dev deps like we do in the MADS repo?

File: components/tabs/src/tabs/tabs.styles.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { makeStyles, shorthands } from '@griffel/react';
 export const useTabsStyles = makeStyles({
   root: {
     color: tokens.colorNeutralForeground1,
+    // @ts-ignore Griffel is not supporting 1 as a value for CSS variables.
```
Comment (brandonthomas) [inline components/tabs/src/tabs/tabs.styles.ts:7]: Does it just want strings

**Review by brandonthomas — APPROVED** [review]

File: components/tabs/src/tabs/tabs.styles.ts (hunk 2 above)
Comment (AkimaLunar) [reply components/tabs/src/tabs/tabs.styles.ts:7]: @brandonthomas, I need to file this on the Griffel team. That tying is incorrect.

**Pull request merged**